src/nexuscli/cli/subcommand_repository.py

```diff
@@ -129,6 +135,47 @@ def cmd_delete(nexus_client, args):
     return errors.CliReturnCode.SUCCESS.value
 
 
+def _cmd_del_assets(nexus_client, repoName, assetRegex, isWildcard, doForce):
+    """Performs ``nexus3 repository delete_assets``"""
+    
+    if not doForce:
+        sys.stdout.write('Retrieving assets matching {}\n'.format(assetRegex))
+        assets_list = nexus_client.repositories.delete_assets(repoName, assetRegex, isWildcard, True)
+        if len(assets_list) == 0:
+            sys.stdout.write('Found 0 matching assets: aborting delete\n')
+            return errors.CliReturnCode.SUCCESS.value
+        
+        sys.stdout.write('Found {} matching assets:\n{}\n'.format(len(assets_list), '\n'.join(assets_list)))
```

Comment:
Since dropping support for Python 2.x, the whole code-base uses f-strings instead of `.format`. Could you please use f-strings for consistency?

Reply:
done!